src/Microsoft.Security.Utilities.Benchmarks/SecretMaskerDetectionBenchmarks.cs

```diff
@@ -52,10 +95,10 @@ private string GeneratePrefix(int size, bool random)
             return prefix;
         }
 
-        private List<string> GenerateExamples(bool random)
+        private List<string> GenerateExamples(bool random, int prefixSize)
```

Comment:
`prefixSizeInBytes` or what?

Reply:
It's in chars. Maybe I should replace size with length throughout?

Reply:
Yes, better.